integrations/extensions/starter-kits/metaweather/readme.md

```diff
@@ -43,9 +43,39 @@ To get started with using the OpenAPI spec, you can upload the `metaweather.adva
       - woeid: 1. OK, I can help with that. What is your ...
       - year: year
       - month: month
+      - date: date
     ```
 If you want to know the Where-On-Earth-ID of a particular location, simply use the first action to query using the location name. The Where-On-Earth-ID of the location will be returned to you.
 
+### Advanced: Blog Spec
+There is [a Medium blog](https://linktoblog) published regarding using this starter kit. The `metaweather-zendesk.blog.actions.json` is a two extension example skill that is highlighted and used in the blog. To configure this spec, you should use the below operations and parameters.
```

Comment:
I'm having a problem parsing: "... published regarding using this starter kit"

Reply:
I think you need to provide here an intro on the fact u can mix & match extensions and then move to the technicalities... at the moment, this feels detached and out of flow